pages/proposals/[proposalId].tsx

```diff
@@ -176,6 +176,15 @@ const Proposal: NextPage = () => {
       <LineAlert msg={initialMessage} variant={initialMessageStatus} />
     ) : null
 
+  let proposalMessageContent = null
+  if (proposal?.msgs?.length) {
```

Comment:
i feel like this would be more clear as a ternary, what do you think? also seems like another place it might make sense to create a component outside of this render, but we can leave it here for now for quicker iteration

Reply:
Definitely moved to a ternary in this case. I do like moving things to sub-components but I wait for "this is feeling gross" because there's some overhead in every separate component file you have to maintain. I think of the proposal component as a "shell" around several other components that reflect the proposal's contents. What were you thinking was a logical sub-group of those?